Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -35,6 +36,7 @@ class BrowserViewController: UIViewController {
     private var homePanelIsInline = false
     private var searchLoader: SearchLoader!
     private var snackBars = UIView()
+    private var auralProgress = AuralProgressBar()
```

Comment:
Nit: Prefer `let` over `var` where possible (while here, `snackBars` should also be defined as a constant).